packages/devtools_app/lib/src/memory/flutter/memory_protocol.dart

```diff
@@ -16,7 +17,7 @@ import 'memory_controller.dart';
 class MemoryTracker {
   MemoryTracker(this.service, this.memoryController);
 
-  static const Duration updateDelay = Duration(milliseconds: 200);
+  static const Duration _updateDelay = Duration(milliseconds: 1000);
```

Comment:
200 ms is too often but I worry that 1 second is too little.  VM seems to report changes about every 400-500 ms would that be a better delay? ADB show changes every 1/2 a second or so.  With any polling we could lose the spike 1 second might be too much.

Reply:
I'll update to 500ms.